### PR TITLE
Improved example lpar_operations

### PR DESCRIPTION
**Pull request by andy-maier**
Please review and merge.

Reason for this change was to have better displaying of the behavior that sometimes after LPAR activation, the status does not update for some time, or not at all.

Details:
 - Added retry loops when retrieving status, until desired status is achieved.
- Changed refresh from pull_full_properties() to using list(), because we just want to use the 'status' property.
- Removed "cpcstatus" input parameter, because when the CPC name is provided, it does not really make sense to filter by status in addition.

### CODE REVIEW

**Comment by andy-maier** [comment]
+1

**Comment by abrezovsky** [comment]
The delay on the updated status is certainly annoying, the solution on my scripts is to sleep for 5 seconds. Your solution is much more elegant.

With the performance issues of pull_full_properties, should we be suggesting using it to just get the status? Considering that something like the following is 150x faster?

`lpars = cec.lpars.list()
for aLpar in lpars:
    if aLpar.name == lparStr:
        status = aLpar.properties['status']
        break
if status == "":
    print("[Error] Lpar has disappeared...")
    sys.exit(1)`

**Comment by andy-maier** [comment]
I agree that using `manager.list()` is preferrable. Will change it.

**Comment by andy-maier** [comment]
I think we could go even further by having the loop on the desired status inside of the `lpar.activate()`, `lpar.deactivate()` and `lpar.load()` methods.

**Comment by abrezovsky** [comment]
The question is, do we want to leave the verification that things went right to the user or handle it ourselves?

**Comment by andy-maier** [comment]
The difficulty with doing it in the zhmcclient layer is async vs. sync. i think that by definition, we cannot do this for async mode, and the user needs to be aware of any deferrals w.r.t. status anyway. So if we cover for that in sync mode, I'm not sure we made it simpler. The sync and async behavior are now more apart than without covering for that.

So i tend to say we should not cover for that, but I'm not entirely sure about it.

File: examples/lpar_operations.py [hunk 1]
```diff
@@ -82,66 +82,89 @@
...
-print("Accessing status of CPC %s ..." % cpc.name)
-status = cpc.get_property('status')
-print("Status of CPC %s: %s" % (cpc.name, status))
-
 print("Finding LPAR by name=%s ..." % lparname)
-try:
-    lpar = cpc.lpars.find(name=lparname)
-except zhmcclient.NotFound:
+# We use list() instead of find() because find(name=..) is optimized by using
+# the name-to-uri cache and therefore returns a Partition object with only a
```
Comment (leopoldjuergen) [inline examples/lpar_operations.py]: Why is the comment about Partitions if we are using LPArs here.

**Comment by leopoldjuergen** [comment]
+1

**Pull request merged**